src/main/antora/modules/security/pages/security.adoc

```diff
@@ -1124,3 +1124,216 @@ The `webClient.addRequestHeader()` calls used here make sure that the headers fo
 If you want to inspect the app yourself, you can manually deploy the WAR file (`security/restCustomAuthCustomStore/target/restCustomAuthCustomStore.war` )to the server of your choice (e.g. GlassFish 7), and request the URL via a browser or a commandline util such as `curl`.
 
 
+
+== Securing an endpoint with Form authentication and remember-me
+
+In the following example, we'll secure a REST endpoint using Form authentication and remember-me.
+
+Remember-me is a facility where an authenticated identity can be remembered beyond the scope of an HTTP session.
```

Comment:
How long? Are there any other features worth mentioning here?

Reply:
Yes, either that or maybe a link to the spec would help, since the spec is readable. That's probably true for some of the other examples, too.